index.bs

```diff
@@ -531,7 +537,6 @@ authorizing an authenticator with which to complete the operation.
     [SecureContext]
     interface ScopedCredentialInfo {
         readonly attribute ScopedCredential     credential;
-        readonly attribute CryptoKey            publicKey;
```

Comment:
Should also delete lines 165-168 since there are no longer any references to CryptoKey or JsonWebKey left in this spec.